modules/aws/bootstrap/main.tf

```diff
@@ -0,0 +1,127 @@
+resource "aws_s3_bucket_object" "ignition" {
+  bucket  = "${var.bucket}"
+  key     = "bootstrap.ign"
+  content = "${var.ignition}"
+  acl     = "private"
+
+  server_side_encryption = "AES256"
+
+  tags = "${var.tags}"
+
+  lifecycle {
+    ignore_changes = ["*"]
+  }
+}
+
+data "ignition_config" "redirect" {
+  replace {
+    source = "s3://${var.bucket}/bootstrap.ign"
+  }
+}
+
+resource "aws_iam_instance_profile" "bootstrap" {
+  name = "${var.cluster_name}-bootstrap-profile"
+
+  role = "${join("|", aws_iam_role.bootstrap.*.name)}"
+
+  #"${var.iam_role == "" ?
```

Comment:
Can this be removed?

Reply:
ah, no, I need to get that working again...  Will hopefully have a fix up shortly ;)

Reply:
> Can this be removed?

Fixed with 1efb239 -> f29ab92.